docs/tutorial/schemas.rst

```diff
@@ -197,8 +205,8 @@ receivers, restart them, and then start using the new version in the publishers.
 You should keep the old schema versions in the schemas package for a reasonable
 amount of time, long enough to make sure all receivers are up-to-date. To avoid
 clutter, we recommend you use a separate module per schema version
-(``yourapp_message_schemas.v1:Message``,
-``yourapp_message_schemas.v2:Message``, etc)
+(``yourapp_message_messages.v1:Message``,
+``yourapp_message_messages.v2:Message``, etc)
```

Comment:
And here (I guess "yourapp_messages.v1:Message", "yourapp_messages.v2:Message").